libraries/Microsoft.Bot.Builder/BotFrameworkAdapter.cs

```diff
@@ -81,13 +83,15 @@ public class BotFrameworkAdapter : BotAdapter, IAdapterIntegration, IUserTokenPr
             RetryPolicy connectorClientRetryPolicy = null,
             HttpClient customHttpClient = null,
             IMiddleware middleware = null,
-            ILogger logger = null)
+            ILogger logger = null,
+            AuthenticationConfiguration authConfig = null)
         {
```

Comment:
This is a breaking change and cannot be made. It's got to be a new constructor.

Reply:
To be fair, it is a binary level breaking change. If people recompile, it will work.
Since it is a patch version I'll add the other constructor, but I don't think we should shackle ourselves in binary compat for minor and major versions. We can take this to the standup :)